yarn.lock

```diff
@@ -437,6 +470,15 @@
     "@babel/helper-skip-transparent-expression-wrappers" "^7.16.0"
     "@babel/plugin-proposal-optional-chaining" "^7.16.7"
 
+"@babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining@^7.17.12":
```

Comment:
you've got two versions of things, need to run deduplicate

Reply:
You're right. fixed